backend/src/errorHandler.ts

```diff
@@ -0,0 +1,28 @@
+import { Logger } from "pino";
+
+import logger from "./logger";
+
+class ErrorHandler {
+    private readonly logger: Logger;
+
+    constructor({ logger }: { logger: Logger }) {
+        this.logger = logger;
+    }
+
+    public async handleExceptionError(error: Error) {
+        this.logger.error(`Uncaught exception: ${error.message}`);
```

Comment:
This will not print backtrace, without backtrace it is about as useless as default handler

Reply:
then probably I will do same as unhandled rejection:
```
[1636714350968] ERROR: Uncaught exception: 
    Error: error message
        at /Users/apple/Documents/subspace-relayer/backend/dist/index.js:120:20
        at Object.<anonymous> (/Users/apple/Documents/subspace-relayer/backend/dist/index.js:133:3)
        at Module._compile (node:internal/modules/cjs/loader:1101:14)
        at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
        at Module.load (node:internal/modules/cjs/loader:981:32)
        at Function.Module._load (node:internal/modules/cjs/loader:822:12)
        at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:79:12)
        at node:internal/main/run_main_module:17:47
```